spl/src/metadata.rs

```diff
@@ -106,18 +106,8 @@ pub fn create_metadata_accounts_v3<'info>(
     ctx: CpiContext<'_, '_, '_, 'info, CreateMetadataAccountsV3<'info>>,
     data: mpl_token_metadata::types::DataV2,
     is_mutable: bool,
-    update_authority_is_signer: bool,
```

Comment:
This argument wasn't being used anywhere.

Reply:
Indeed, but we need to pass this information to the instruction. This will likely required a tweak on the Rust client, I will work on it.